<launch>
    <arg name="visualization" default="true"/>
    <!-- <arg name="location" default="dischma_valley"/> -->
    <arg name="location" default="duebendorf"/>

    <node pkg="tf" type="static_transform_publisher" name="world_map" args="0 0 0 0 0 0 world map 10"/>

    <node pkg="terrain_abort_planner" type="run_dynamic_rallypoints" name="rrt_planner" output="screen">
        <param name="location" value="$(arg location)"/>
        <param name="map_path" value="$(find terrain_abort_planner)/resources/$(arg location).tif"/>
        <param name="mission_file_path" value="$(find terrain_abort_planner)/resources/mission.plan"/>
        <param name="color_file_path" value="$(find terrain_abort_planner)/resources/$(arg location)_color.tif"/>
        <param name="output_directory" value="$(find terrain_abort_planner)/../output"/>
    </node>
    <group if="$(arg visualization)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find terrain_abort_planner)/launch/rallypoints.rviz" />
    </group>
</launch>
